routers/web/org/members.go

```diff
@@ -44,9 +44,9 @@ func Members(ctx *context.Context) {
 			ctx.Error(http.StatusInternalServerError, "IsOrgMember")
 			return
 		}
-		opts.PublicOnly = !isMember && !ctx.Doer.IsAdmin
+		opts.IsMember = isMember
```

Comment:
You can inline this line now into `#42`

Reply:
@delvh not directly as i then would have to declare `var err error` before instead so you dont save anything